src/components/Form/TutoringPriceQuote.js

```diff
@@ -257,12 +288,16 @@ const TutoringPriceQuote = ({courseType, handleUpdatePriceFields, tuitionConfirm
                                 </Select>
                             </Grid>
                             <Grid item xs={3}>
-                                <TextField
-                                    label={"Hourly Tuition"}
-                                    value={hourlyTuition || ''}
-                                    InputProps={{readOnly:true}}
-                                    variant={"outlined"}
-                                />
+                                    <InputLabel htmlFor={"hourly-tuition"}>Hourly Tuition</InputLabel>
+                                <div
+                                    style={{
+                                        padding: '10px 26px 10px 12px',
+                                    }}
+                                >
+                                    <Typography variant="h6">
+                                        {hourlyTuition || ''}
```

Comment:
Unnecessary `||`, if `hourlyTuition` is undefined or null, nothing will be rendered regardless.